crypto/signing/mcl/multisig/bls.go

```diff
@@ -281,8 +337,10 @@ func hashPublicKeyPoint(hasher hashing.Hasher, pubKeyPoint crypto.Point) ([]byte
 		return nil, err
 	}
 
-	// H1(pubkey_i)
-	h := hasher.Compute(string(pointBytes))
+	concatPkWithPKs := append(concatPubKeys, pointBytes...)
```

Comment:
Shouldn't it be the other way around? pki, {pk1, pk2....}

Reply:
Although it shouldn't matter, indeed the specification is pki, {pk1,...pkn}
done